Makefile

```diff
@@ -11,7 +11,6 @@ ENVTEST_K8S_VERSION = 1.26.1
 PLATFORM=$(shell uname -s | tr '[:upper:]' '[:lower:]')
 ARCH=$(shell uname -m | sed 's/x86_64/amd64/')
 
-PROJECT_ID := $(shell gcloud config get-value project)
```

Comment:
@nstogner see it's no longer set at the top level. that's part of the improvement of this PR